doc/user/content/get-started.md

```diff
@@ -5,216 +5,271 @@ menu: "main"
+{{< tab "Linux">}}
 
-    ```sql
-    SELECT * FROM pseudo_source;
-    ```
-    ```nofmt
-     key | value
-    -----+-------
-     a   |     1
-     a   |     2
-     a   |     3
-     a   |     4
-     b   |     5
-     c   |     6
-     c   |     7
+1. Assuming you're using [apt](https://linuxize.com/post/how-to-use-apt-command/), open a terminal and run (as _root_):
+
+    ```shell
+    # Add the signing key for the Materialize apt repository
+    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 79DEC5E1B7AE7694
+    # Add and update the repository
+    sh -c 'echo "deb http://apt.materialize.com/ generic main" > /etc/apt/sources.list.d/materialize.list'
+    apt update
+    # Install materialized
+    apt install materialized
     ```
 
-1. Determine the sum of the values for each key:
+    **Note:** For a _curl_-based alternative, see [Install](/install/#curl-1).
 
-    ```sql
-    SELECT key, sum(value) FROM pseudo_source GROUP BY key;
+1. Once the installation is complete, you can start the `materialized` process:
+
+    ```shell
+    materialized -w 1
     ```
-    ```nofmt
-     key | sum
-    -----+-----
-     a   |  10
-     b   |   5
-     c   |  13
+
+    This starts a process using one [worker thread](/connect/cli//#worker-threads) and listening on port 6875 by default.
+
+1. You can then connect to the running instance using any [Materialize-compatible CLI](/connect/cli/), like `psql` or `mzcli`. If you have `psql` installed on your machine, connect using:
+
+    ```shell
+    psql -U materialize -h localhost -p 6875 materialize
     ```
 
-    We can actually then save this query as its own materialized view:
+    Otherwise, you can find the steps to install and use your CLI of choice under [Install](/install/#cli-connections).
+
+{{< /tab >}}
+
+{{< /tabs >}}
+
+## Explore a streaming source
+
+Materialize allows you to work with streaming data from multiple external sources using nothing but standard SQL. You write arbitrarily complex queries, Materialize takes care of maintaining the results automatically up to date with very low latency.
+
+We'll start with some sample real-time data from a [PubNub stream](https://www.pubnub.com/developers/realtime-data-streams/) receiving the latest market orders for a given marketplace.
+
+1. Let's create a [PubNub source](/sql/create-source/json-pubnub/#pubnub-source-details) that connects to the market orders channel with a subscribe key:
 
     ```sql
-    CREATE MATERIALIZED VIEW key_sums AS
-        SELECT key, sum(value) FROM pseudo_source GROUP BY key;
+    CREATE SOURCE market_orders_raw 
+    FROM PUBNUB
+    SUBSCRIBE KEY 'sub-c-4377ab04-f100-11e3-bffd-02ee2ddab7fe'
+    CHANNEL 'pubnub-market-orders';
     ```
 
-1. Determine the sum of all keys' sums:
+    The `CREATE SOURCE` statement is a definition of where to find and how to connect to our data source — Materialize won't start ingesting data just yet.
+
+    To list the columns created:
```

Comment:
I would break the column listing into its step.

Reply:
Since I'm not really going into the output of this statement, it's more of a "execute if you're curious" than a real step.